records_mover/session.py

```diff
@@ -36,11 +37,26 @@ def _infer_session_type() -> str:
 
 
 def _infer_scratch_s3_url(session_type: str) -> Optional[str]:
-    scratch_s3_url: Optional[str]
-
     if "SCRATCH_S3_URL" in os.environ:
         return os.environ["SCRATCH_S3_URL"]
 
+    # config_resolver logs at the WARNING level for each time it
+    # attempts to load a config file and doesn't find it - which given
+    # it searches a variety of places, is quite noisy.
+    #
+    # https://github.com/exhuma/config_resolver/blob/master/doc/intro.rst#logging
+    #
+    # https://github.com/exhuma/config_resolver/issues/69
```

Comment:
I have a PR open - hopefully it gets accepted: https://github.com/exhuma/config_resolver/pull/70

Reply:
Should be fine now. Let me know if there are any remaining issues.